Marlin/stepper.cpp

```diff
@@ -788,6 +804,13 @@ void Stepper::isr() {
 
   // If current block is finished, reset pointer
   if (all_steps_done) {
+    if(IS_CHUNK(current_block)) {
+      //sync planner position back up with stepper positions
+      Planner::sync_from_steppers();
+      //temperature ISR can get drowned out under high step rate, make sure it gets run
+      Temperature::isr();
```

Comment:
did not actually test it with the temperature ISR in here. should only be needed at high step rates anyways?